browser/components/shared/button/ButtonGroup.js

```diff
@@ -5,11 +5,15 @@ import PropTypes from 'prop-types';
 ButtonGroup.propTypes = {
     text: PropTypes.string,
     align: PropTypes.string,
+    marginTop: PropTypes.number,
```

Comment:
Exposing styling attributes is a never ending road that leads to a bloated api for the component :-)
Each component should have a default style/layout, but styling based on the context it is used in, should be handled by the parent.
https://stackoverflow.com/questions/49619754/how-to-style-child-components-in-react-styled-components